CREDITS.md

```diff
@@ -55,11 +57,13 @@ Contributors to the Firecracker repository:
 * Hermes <hermes.espinola@gmail.com>
 * Iggy Jackson <iggy@theiggy.com>
 * Ishwor Gurung <me@ishworgurung.com>
+* iulianbarbu <iul@amazon.com>
```

Comment:
```suggestion
*Iulian Barbu <iul@amazon.com>
```

Reply:
The list is automatically generated at every release from the commit author. Manually updating it won't fix the problem.

CC: @firecracker-microvm/compute-capsule do we have a mechanism to change a contributors name?

Reply:
This is part of the default `git log` feature set, and we already implement [it](https://github.com/firecracker-microvm/firecracker/blob/master/.mailmap). This is also where we should deduplicate Petre, by the way.

Reply:
Alias `iulianbarbu` is taken from my git repository configuration. I updated my configuration to `Iulian Barbu`. I think this issue will be fixed with the following contributions, and in the next release my name will apppear as @sandreim suggested.

Reply:
Added it to the mailmap.